presto-orc/src/main/java/com/facebook/presto/orc/OrcWriteValidation.java

```diff
@@ -558,6 +559,16 @@ private static long hashPositionSkipNullMapKeys(Type type, Block block, int posi
                 }
             }
 
+            if (type.getTypeSignature().getBase().equals(StandardTypes.TIMESTAMP_MICROSECONDS)) {
```

Comment:
I don't see any test cases covering this block for Milliseconds. I see such values are omitted from testing in a few places to avoid this.

https://github.com/prestodb/presto/blob/c5c0866608ef4c779e2346e18f2e5aa10f6c92ee/presto-orc/src/test/java/com/facebook/presto/orc/TestSelectiveOrcReader.java#L269-L272

https://github.com/prestodb/presto/blob/313c2d98cf6a80f273b3565150b8218b037913f8/presto-orc/src/test/java/com/facebook/presto/orc/AbstractTestOrcReader.java#L140-L143